scripts/package-distribution.ps1

```diff
@@ -167,17 +180,13 @@ foreach ($Build in $AllBuilds) {
         if ($BuildBaseName -like "*openssl3*") {
             $Tls = "openssl3"
         }
-        $UbuntuVersion = "none"
-        $match = [regex]::Match($BuildBaseName, "ubuntu_(\d{4})")
-        if ($match.Success) {
-            $UbuntuVersion = $match.Groups[1].Value
-        }
+
         if ($BuildBaseName -like "*arm64_*") {
-            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm64 --tls $Tls --ubuntu $UbuntuVersion
+            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm64 --tls $Tls --xdp $UseXdp --time64 $Time64Distro
         } elseif ($BuildBaseName -like "*arm_*") {
-            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm --tls $Tls --ubuntu $UbuntuVersion
+            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm --tls $Tls --xdp $UseXdp --time64 $Time64Distro
         } else {
-            & $RootDir/scripts/make-packages.sh --output $DistDir --tls $Tls --ubuntu $UbuntuVersion # x64
+            & $RootDir/scripts/make-packages.sh --output $DistDir --tls $Tls --xdp $UseXdp --time64 $Time64Distro # x64
```

Comment:
Honestly, I don't see a fundamental difference between UbuntuVersion when it's ubuntu and UseXdp and Time64Distro. They are basically the same thing and make-packages.sh actually assumes that XDP is ubuntu 24.04.

They knobs would make sense if you pass the correct value to them from here.

For example, for arm, we don't pass `--xdp` but we pass `--time64` because we don't build xdp for arm but if `$Time64Distro` is true, we use the t64 version of libssl3.

The current code basically creates 2 aliases for ubuntu24.04.